src/Data/Aeson/BetterErrors/Internal.hs

```diff
@@ -296,6 +296,10 @@ as :: (Functor m, Monad m) => (A.Value -> Maybe a) -> JSONType -> ParseT err m a
 as pat ty = liftParse $ \v ->
   maybe (Left (WrongType ty v)) Right (pat v)
 
+-- | Return the current JSON 'A.Value' as is.
```

Comment:
Perhaps we should also note that it's better to use other functions where possible, like we do with `asArray` or `asObject`?